src/compas/geometry/plane.py

```diff
@@ -384,9 +384,15 @@ def is_parallel(self, other, tol=None):
         >>> plane2 = Plane([1.0, 1.0, 1.0], [0.0, 0.0, 1.0])
         >>> plane1.is_parallel(plane2)
         True
+        >>> plane1 = Plane.worldXY()
+        >>> plane2 = Plane([1.0, 1.0, 1.0], [0.0, 0.0, -1.0])
+        >>> plane1.is_parallel(plane2)
+        True
 
         """
-        return TOL.is_close(self.normal.dot(other.normal), 1, rtol=0, atol=tol)
+        is_parallel = TOL.is_close(self.normal.dot(other.normal), 1, rtol=0, atol=tol)
```

Comment:
why not just take the absolute value of the dot product?

Reply:
agreed. I've made the changes.